lib/rest2.js

```diff
@@ -1750,11 +1769,11 @@ class RESTv2 {
    * @param {number} params.isPublic    - make Pulse message public
    * @param {string} params.isPin       - make Pulse message pinned
    * @param {string} params.attachments - Base64 encoded list of strings
-   * @param {Function} [cb]             - callback
+   * @param {Function} [cb]             - legacy callback
    * @returns {Promise} p
    * @see https://docs.bitfinex.com/reference#rest-auth-pulse-add
    */
-  addPulseComment (params, cb) {
+  addPulseComment (params, cb = null) {
```

Comment:
Weird that this method calls addPulse with same params but the docs does not match, this one has params.parent and addPulse has params.title

Reply:
I think title is required only when parent is missing